beautify.js

```diff
@@ -50,14 +57,36 @@ const beautifyText = async () => {
         i++;
     }
 
-    if (lineCount === 1 || lineCount === 2)
+    if (centerVertically && (lineCount === 1 || lineCount === 2)) {
         output.splice(0, 0, '\n');
+        lineCount++;
+    }
+
+    if (newlinesFill && lineCount < 4) {
+        output.splice(output.length, 0, '\n'.repeat(4 - lineCount));
+    }
 
     text.value = output.join('');
     await updateText(text.value);
 }
 
-const initBeautifyButton = () => {
+const initBeautify = async () => {
     const button = document.querySelector('#beautify-button');
     button.addEventListener('click', beautifyText);
+
+    const CVInput = document.querySelector('#center-vertically');
+    const NFInput = document.querySelector('#newlines-fill');
+
+    const { centerVertically, newlinesFill } = await browser.storage.local.get(['centerVertically', 'newlinesFill']);
+
+    CVInput.checked = centerVertically === undefined ? true : centerVertically;
```

Comment:
Don't default it to true
```suggestion
    CVInput.checked = centerVertically;
```